backend/api/Services/MapService.cs

```diff
@@ -136,7 +121,7 @@ private async Task<Image> DownloadMapImageFromBlobStorage(Mission currentMission
             );
             var blobClient = blobContainerClient.GetBlobClient(currentMission.Map.MapName);
 
-            using var stream = await blobClient.OpenReadAsync();
+            await using var stream = await blobClient.OpenReadAsync();
```

Comment:
I'm not familiar with using await in front here. Is this correct?

Reply:
Resharper suggested it

Reply:
https://stackoverflow.com/questions/58791938/c-sharp-8-understanding-await-using-syntax
"Similar as **using (...)** uses `IDisposable` to clean up resources, **await using (...)** uses `[IAsyncDisposable]`(https://learn.microsoft.com/en-us/dotnet/api/system.iasyncdisposable?view=dotnet-plat-ext-3.0). This allows to perform also time-consuming tasks (e.g involving I/O) on cleanup without blocking."